client/src/components/avatar/Avatar.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import classnames from "classnames";
+
+import "./avatar.css";
+
+type AvatarProps = {
+    className?: string;
+    src: string;
+    size: "small" | "medium" | "large";
+};
+
+export const Avatar = (props: AvatarProps) => {
+    const { className, size } = props;
+    const [path, setPath] = useState(props.src);
+    const classNames = classnames("avatar", `avatar_size-${size}`, className);
+
+    const onError = () => {
+        setPath("images/avatar-default.png");
+    };
```

Comment:
Для хендлеров нужно использовать хук `useCallback`

```suggestion
    const onError = useCallback(() => { setPath("images/avatar-default.png"); }, []);
```

Тогда он не будет пересоздаваться при каждом рендере компонента